src/site/content/en/fast/reduce-network-payloads-using-text-compression/index.md

```diff
@@ -156,3 +158,35 @@ const app = express();
 
 app.use(express.static('public'));
 </pre>
+
+## Effects on Core Web Vitals
+
+Depending the type of resource, both compression and minification can have a variety of effects on Core Web Vitals. Since both minification and compression reduce the payload size of the delivered resource, it's important to understand what Core Web Vitals are affected by these performance optimizations.
```

Comment:
This makes it sound like it has good and bad effects. Surely it's pretty much always good? So should it be more "can positively impact Core Web Vitals" or something like that.

Also I think there should be a generic comment in this section about the fact smaller payloads, reduces network contention, which improves loading performance.